Stream_support/include/CGAL/IO/Color.h

```diff
@@ -61,6 +61,12 @@ class Color {
     return !( (*this) == c);
   }
 
+#ifndef CGAL_CFG_NO_CPP0X_DELETED_AND_DEFAULT_FUNCTIONS
```

Comment:
For some reason, this breaks on MSVC 2013. See tfor example  [here](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.14-Ic-44/AABB_tree/TestReport_afabri_x64_Cygwin-Windows10_MSVC2013-Debug-64bits.gz)

Reply:
That is the Boost issue https://svn.boost.org/trac10/ticket/13396 fixed only in version 1.67 or later.

The definition of our macro `CGAL_CFG_NO_CPP0X_DELETED_AND_DEFAULT_FUNCTIONS` should:
  - check the new macro `BOOST_NO_CXX11_DEFAULTED_MOVES`,
  - *and* check `_MSC_VER` also.

Reply:
My patch (untested):
```patch
diff --git a/Installation/include/CGAL/config.h b/Installation/include/CGAL/config.h
index 0c1d7a3cbb9..bed2e36d452 100644
--- a/Installation/include/CGAL/config.h
+++ b/Installation/include/CGAL/config.h
@@ -210,7 +210,8 @@
 #if defined(BOOST_NO_DELETED_FUNCTIONS) || \
     defined(BOOST_NO_DEFAULTED_FUNCTIONS) || \
     defined(BOOST_NO_CXX11_DELETED_FUNCTIONS) || \
-    defined(BOOST_NO_CXX11_DEFAULTED_FUNCTIONS) || (BOOST_VERSION < 103600)
+    defined(BOOST_NO_CXX11_DEFAULTED_FUNCTIONS) || (BOOST_VERSION < 103600) || \
+   (defined(_MSC_VER) && _MSC_VER < 1900) // MSVC 2013 has only partial support
 #define CGAL_CFG_NO_CPP0X_DELETED_AND_DEFAULT_FUNCTIONS 1
 #endif
 #if defined(BOOST_NO_FUNCTION_TEMPLATE_DEFAULT_ARGS) || \
```

@barcode Can you please integrate that patch in your pull-request? Or declare one of us as collaborator on https://github.com/barcode/cgal?